### PR TITLE
M2 Math magicians: components

### PR DESCRIPTION
**Pull request by ShiroYaksha90**
- Delete all the boilerplate from Create React App (text, images, styles).
- Create a new directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:
calculator

- Use functional components
- Create at least 1 child component
- Import and use it in the top-level component: App.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,147 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Ghiftee) [inline README.md]: - [ ] Kindly fill out the ```acknowledgement``` section of your README
- [ ] Also, your ```MIT license``` is a broken link. Kindly fix it
These would make your project documentation more professional

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './calculator.css';
+
+function Calculatore() {
```
Comment (Ghiftee) [inline src/components/calculator.js:6]: - [ ] According to the requirements, your ```Calculator``` function should have at least one child component. At the moment, your ```Calculator``` component only contains a tree of DOM elements (in this case, div and button elements) and does not render any other React components inside of it.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @ShiroYaksha90  👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ No linter errors
✔️ Components are created
✔️ App matches given design

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._
- [ ] There are still some CRA boilerplate files left in your project ```public folder```. Kindly ensure all of them are removed. The ```logo.svg``` file can also be deleted.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by ShiroYaksha90** [comment]
@Ghiftee Thank you I implemented the required changes

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @ShiroYaksha90,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**